apps/mark-scan/frontend/src/pages/ask_poll_worker_page.tsx

```diff
@@ -17,16 +15,17 @@ interface Props {
 // 1. Voter stage: tells the voter to ask a poll worker for help. Does not allow the voter to advance the state.
 // 2. Poll worker stage: after auth, gives instructions and a button to advance the state.
 export function AskPollWorkerPage(props: Props): JSX.Element {
-  const { authStatus, children, pollWorkerPage } = props;
+  const { authStatus, children, hideTitle, pollWorkerPage } = props;
```

Comment:
Supporting hidden titles allows us to render `Icon`s (or any other element) above the title. Without this change the `Icon` is rendered between the title and body which looks awkward.